web/samples/chirality/chirality.js

```diff
@@ -0,0 +1,69 @@
+            'leftctrl': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "",  "", "ʍ", "ə", "ɹ", "ð", "ʏ", "ʉ", "ɨ", "ɵ", "ʘ", "",  "",  "",   "", "", "", "ɑ", "ʃ", "",  "ɸ", "ɣ", "ɥ", "ɟ", "",  "ɬ", "ː", "",   "", "", "", "", "", "", "ʒ", "χ", "ç", "ʋ", "β", "ɲ", "",  "",  "",  "",  "", "", "", "", "", ""),
+      'leftctrl-shift': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "ʡ", "",  "ɘ", "ʀ", "θ", "",  "ɯ", "ɪ", "",  "",  "",  "",  "",   "", "", "", "ɒ", "ᶘ", "",  "",  "ɢ", "ʜ", "",  "",  "ʟ", "ː", "",   "", "", "", "", "", "", "ᶚ", "",  "",  "",  "ʙ", "ɴ", "",  "",  "",  "ʔ", "", "", "", "", "", ""),
+             'leftalt': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "",  "", "", "ɛ", "ɽ", "ʈ", "ɥ", "ʊ", "",  "ɔ", "",  "",  "",  "",   "", "", "", "æ", "",  "ɖ", "",   "", "ɦ", "ʝ", "",  "ɭ", "",  "",   "", "", "", "", "", "", "ʐ", "",  "ɕ", "", "",  "ɳ", "",  "",  "",  "",  "", "", "", "", "", ""),
+            'rightalt': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "ʠ", "ɰ", "ɜ", "ɾ", "ƭ", "",  "ʌ", "",  "ø", "ƥ", "",  "",  "",   "", "", "", "ɐ", "σ", "ɗ", "",  "ɠ", "ħ", "ʄ", "ƙ", "ɮ", "",  "",   "", "", "", "", "", "", "ʑ", "",  "ƈ", "",  "ɓ", "ŋ", "ɱ", "",  "",  "",  "", "", "", "", "", ""),
+       'leftalt-shift': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "ʢ", "",  "œ", "ɻ", "",  "",  "",  "ᵻ", "ɞ", "",  "",  "",  "",   "", "", "", "",  "",  "",  "",  "",  "",  "",  "",  "ʎ", "",  "",   "", "", "", "", "", "", "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "", "", ""),
+      'rightalt-shift': new Array("",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "",  "",  "ɶ", "ʁ", "",  "",  "ᵾ", "ᵼ", "ɤ", "",  "",  "",  "",   "", "", "", "ᴂ", "",  "",  "",  "ʛ", "ɧ", "",  "",  "ɺ", "",  "",   "", "", "", "", "", "", "",  "",  "",  "",  "",  "",  "",  "",  "",  "",  "", "", "", "", "", "")   
+      }
+  };
+  this.KH = '';
+  this.KM = 0;
+
+  this.dfltCodes = ["K_BKQUOTE","K_1","K_2","K_3","K_4","K_5","K_6","K_7","K_8","K_9","K_0",
+      "K_HYPHEN","K_EQUAL","K_*","K_*","K_*","K_Q","K_W","K_E","K_R","K_T",
+      "K_Y","K_U","K_I","K_O","K_P","K_LBRKT","K_RBRKT","K_BKSLASH","K_*",
+      "K_*","K_*","K_A","K_S","K_D","K_F","K_G","K_H","K_J","K_K","K_L",
+      "K_COLON","K_QUOTE","K_*","K_*","K_*","K_*","K_*","K_oE2",
+      "K_Z","K_X","K_C","K_V","K_B","K_N","K_M","K_COMMA","K_PERIOD",
+      "K_SLASH","K_*","K_*","K_*","K_*","K_*","K_SPACE"];
+  
+  this.gs = function (t, e) {
+      return this.g0(t, e);
+  };
+  this.g0 = function (t, e) {
+    var k = KeymanWeb, r = 0, m = 0;
+    
+    // Handwritten time!
+    var kls = this.KV.KLS;
+    
+    var layers = ['leftctrl', 'default', 'shift', 'leftctrl', 'leftctrl-shift', 'leftalt', 'rightalt', 'leftalt-shift', 'rightalt-shift'];
```

Comment:
The first `leftctrl` should be removed to line up with the layers defined in `this.KV.KLS`

Reply:
Oh, right.  There was a point in debugging where I suspected it was only loading the first layer or two and duplicated it there to test my theory - forgot to undo it.  Thanks for pointing it out!